### ISSUE TITLE
buildSFX() no longer works...

### ISSUE THREAD

**Issue by lookfirst** [systemjs/builder#171]
All of the sudden, buildSFX() just up and stopped working correctly for me.

It is now including a file at the top of my bundle that does not have `System.import()` in it. It looks like the contents of `builder/templates/sfx-core.js`.

```
jspm --version 0.15.6
"systemjs-builder": "^0.10.6"
es6-module-loader@0.16.6
systemjs@0.16.11
"postinstall": "jspm install && jspm dl-loader",
```

**Comment by guybedford** [comment]
@lookfirst sfx bundles don't support System.import as that implies dynamic loading, which then just becomes a normal bundle. Just let me know if you have further questions at all.

**Comment by lookfirst** [comment]
I want dynamic loading of css files.

**Comment by guybedford** [comment]
Right - so any type of dynamic loading means using the full SystemJS with a normal bundle.

**Comment by lookfirst** [comment]
It is quite unclear that there is a 'full SystemJS' and a 'partial SystemJS' that is only for SFX bundles.

From the docs: `To make a bundle that is independent of the SystemJS loader entirely, we can make SFX bundles:`

That said, when I encountered this problem today, I also tried to revert my code back to just a simple `build()` and append es6 loader and system.js and got all sorts of weird errors there too. Something is really fishy here.
